autometa/taxonomy/ncbi.py

```diff
@@ -167,117 +159,123 @@ def lineage(self, taxid, canonical=True):
                 'name': self.name(taxid),
                 'rank': self.rank(taxid)})
             taxid = self.parent(taxid)
+        lineage.append({  # adds root
+            'taxid': taxid,
+            'name': self.name(taxid),
+            'rank': self.rank(taxid)})
```

Comment:
Do we want to add the root to every lineage? We can assume _all_ taxids have root in their lineage. Is there a justification to adding the root to _every_ taxids lineage?